lib/utils/couchclient/src/index.js

```diff
@@ -107,6 +107,45 @@ const key = (uri) => {
   return undefined;
 };
 
+// List all databases on the specified server
```

Comment:
It's there because this is the order in the mongoclient. And in the mongoclient, the order is such because these methods are also used elsewhere (e.g. findServer).